pkg/appconsts/global_consts.go

```diff
@@ -11,7 +13,8 @@ import (
 // They can not change throughout the lifetime of a network.
 const (
 	// NamespaveVersionSize is the size of a namespace version in bytes.
-	NamespaceVersionSize = 1
+	NamespaceVersionSize     = 1
+	NamespaceVersionMaxValue = math.MaxUint8 // this value should be set according to NamespaceVersionSize, i.e., 2^NamespaceVersionSize - 1
```

Comment:
I think there's a slight typo in `2^NamespaceVersionSize - 1` b/c it doesn't evaluate to `math.MaxUint8`. When `NamespaceVersionSize` is interpolated, this evaluates to `2 ^ 1 - 1`. Maybe this meant to say something like `... i.e., // math.Pow(2, 8 * NamespaceVersionSize) - 1` but proposal to do this instead:

```go
	// NamespaceVersionMaxValue is the maximum value a namespace version can be.
	// This const must be updated if NamespaceVersionSize is changed.
	NamespaceVersionMaxValue = math.MaxUint8
```

Reply:
You are right, when writing that formula, I was thinking of `NamespaceVersionSize` in bits, instead of bytes. The correction is as you suggested `2^(8* NamespaceVersionSize)-1)`. Going to fix it.

Reply:
Done in https://github.com/celestiaorg/celestia-app/pull/1893/commits/56a8fd21253afa4f986c5fb1d83ab58680a75b82.